contracts/contracts/Poll.sol

```diff
@@ -3,18 +3,18 @@ pragma solidity ^0.8.10;
 
 import {IMACI} from "./IMACI.sol";
 import {Params} from "./Params.sol";
-import {Hasher} from "./crypto/Hasher.sol";
-import {Verifier} from "./crypto/Verifier.sol";
 import {SnarkCommon} from "./crypto/SnarkCommon.sol";
-import {SnarkConstants} from "./crypto/SnarkConstants.sol";
 import {DomainObjs, IPubKey, IMessage} from "./DomainObjs.sol";
 import {AccQueue, AccQueueQuinaryMaci} from "./trees/AccQueue.sol";
 import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
 import {ERC20} from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
 import {SafeERC20} from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
 import {VkRegistry} from "./VkRegistry.sol";
+import {Verifier} from "./crypto/Verifier.sol";
 import {EmptyBallotRoots} from "./trees/EmptyBallotRoots.sol";
 import {TopupCredit} from "./TopupCredit.sol";
+import {Utilities} from "./Utility.sol";
+import {MessageProcessor} from "./MessageProcessor.sol";
```

Comment:
this is not used anywhere

Reply:
it is used in contract Poll (the inheritance part)